public/js/reader.js

```diff
@@ -111,33 +111,27 @@ Reader.loadImages = function () {
             if (Reader.infiniteScroll) {
                 Reader.initInfiniteScrollView();
             } else {
-                $(window).on('wheel', function(e) {
+                $(window).on("wheel", function(e) {
                     if(Reader.isFullscreen) {
-                        if (e.originalEvent.deltaY > 0) Reader.changePage(-1);
-                        else Reader.changePage(1);
+                        let changePage_val = 1;
+                        if (e.originalEvent.deltaY > 0) changePage_val = -1;
+                        if (!Reader.mangaMode) changePage_val *= -1;
+                        Reader.changePage(changePage_val);
                         return false;
                     }
                 });
-                $(document).on('click', function(event) {
-                    var img_y_offset = $('a#display img').offset()['top'] + $('a#display img').height();
-                    if (event.pageY < img_y_offset && !$('#overlay-shade').is(':visible') ){
-                        var whereside = event.pageX < $(window).width()/2 ? 'left' : 'right';
-                        if (!$(event.target).closest('a#display').length) {
-                            whereside = 'outer-'+whereside;
-                        }
-                        switch (whereside) {
-                        case "outer-left":
-                            Reader.changePage("first");
-                            break;
+                // when click left or right img area change page
+                $(document).on("click", function(event) {
+                    // check click Y position is in img Y area 
+                    if (event.pageY < ($("a#display img").offset()["top"] + $("a#display img").height()) && !$("#overlay-shade").is(":visible") ){
```

Comment:
This needs a check for `event.pageY > [height of header]` in case the header is visible.

Reply:
Also, clicking the arrow buttons triggers this and causes the page to switch twice. 🥲

Reply:
maybe if we check height of header then button trigger will be fix

Reply:
if we don't need outer click just change if check part